capa/features/extractors/binja/extractor.py

```diff
@@ -45,9 +54,58 @@ def extract_global_features(self):
     def extract_file_features(self):
         yield from capa.features.extractors.binja.file.extract_features(self.bv)
 
+    def _build_call_graph(self):
```

Comment:
@xusheng6 can you think of a way to compute the call graph without accessing the LLIL? This would save another 15s (5%) of runtime if we didn't have to enumerate all the instructions.

Perhaps we could just use `function.callers`?

Reply:
`function.callers` doesn't filter the references by calls, just any code reference to this function. maybe this is ok. we should investigate if this loss in precision is meaningful or not.